From 5b6c2459b57c23669d6efc5a14a874c693004eec Mon Sep 17 00:00:00 2001
From: Protobuf Team Bot <protobuf-github-bot@google.com>
Date: Thu, 14 Sep 2023 12:04:25 -0700
Subject: [PATCH] Expose alias to protobuf_nowkt runtime

This will allow users such as Kythe to use the same runtime as the default toolchain in their own proto_lang_toolchain definitions.

PiperOrigin-RevId: 565438071
---
 BUILD.bazel | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/BUILD.bazel b/BUILD.bazel
index 88a19996826..492b750c8a3 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -198,6 +198,14 @@ cc_binary(
 # C++ runtime
 ################################################################################

+# Expose the runtime for the proto_lang_toolchain so that it can also be used in
+# a user-defined proto_lang_toolchain.
+alias(
+  name = "protobuf_nowkt",
+  actual = "//src/google/protobuf:protobuf_nowkt",
+  visibility = ["//visibility:public"],
+)
+
 # The "lite" runtime works for .proto files that specify the option:
 #     optimize_for = LITE_RUNTIME;
 #
